src/core_plugins/kibana/index.js

```diff
@@ -66,6 +66,7 @@ export default function (kibana) {
         listed: false,
         description: 'the kibana you know and love',
         main: 'plugins/kibana/kibana',
+        styleSheetPath: `${__dirname}/public/index.scss`,
```

Comment:
`scss` or `css`?

Reply:
The examples @tylersmalley provided used scss here, but I think it accepts any. Basically the plugin needs a pointer to the root file it needs to compile.